doc/api/cli.md

```diff
@@ -88,6 +88,15 @@ added: v10.5.0
 
 Enable experimental worker threads using the `worker_threads` module.
 
+### `--expose-internals, --expose_internals`
```

Comment:
Do not include the `_` variant. All flags that have `-` can use `_` instead. (Perhaps that fact needs to be documented somewhere, but if so, that should be in a separate pull request.)